Change needed to get stdbuf to successfully preload libstdbuf.so
for both 32-bit and 64-bit applications.

This change has been passed upstream.

--- coreutils-8.16/src/stdbuf.c.orig	2014-05-12 13:48:24.644347899 -0700
+++ coreutils-8.16/src/stdbuf.c	2014-05-14 09:45:52.507614923 -0700
@@ -200,6 +200,14 @@
   char const *const search_path[] = {
     program_path,
     PKGLIBEXECDIR,
+
+   /* Since we don't link against libstdbuf.so, we need to LD_PRELOAD it.
+      libstdbuf.so lives in /usr/lib and /usr/lib/64, so we just use a
+      search path value of "" and that equates to LD_PRELOAD=libstdbuf.so,
+      and ld.so.1 finds the right one associated with the process class.
+      ie. we don't use full path names for the preload names.  */
+
+    "",              /* System default */
     NULL
   };
 
